ibm/service/vpc/resource_ibm_is_security_group.go

```diff
@@ -410,7 +417,19 @@ func resourceIBMISSecurityGroupDelete(d *schema.ResourceData, meta interface{})
 				deleteSecurityGroupTargetBindingOptions := sess.NewDeleteSecurityGroupTargetBindingOptions(id, *securityGroupTargetReference.ID)
 				response, err = sess.DeleteSecurityGroupTargetBinding(deleteSecurityGroupTargetBindingOptions)
 				if err != nil {
-					return fmt.Errorf("[ERROR] Error deleting security group target binding while deleting security group : %s\n%s", err, response)
+					if response != nil {
+						if response.StatusCode == 404 {
+							log.Printf("[DEBUG] Security group target(%s) binding is already deleted", *securityGroupTargetReference.ID)
+						} else if response.StatusCode == 409 {
+							log.Printf("[DEBUG] Security group target(%s) binding is in deleting, will wait till target is removed", *securityGroupTargetReference.ID)
```

Comment:
[DEBUG] Security group target(%s) binding is in deleting status, waiting till target is removed